src/heatmapFeature.js

```diff
@@ -26,6 +26,11 @@ var transform = require('./transform');
  *   be computed.
  * @param {number} [updateDelay=1000] Delay in milliseconds after a zoom,
  *   rotate, or pan event before recomputing the heatmap.
+ * @param {boolean|number|'auto'} [binned='auto'] If true or a number,
```

Comment:
I agree -- either would be fine as the default.  `auto` definitely introduces a discontinuity in the visualization as the number of points crosses the threshold (resizing the window could also do this).  On the other hand, if you have a million data points, binning will be much faster.  Your call on which should be the default.